Deployment/Jenkins/master/parent-instance.groovy

```diff
@@ -0,0 +1,303 @@
+pipeline {
+
+    agent any
+
+    options {
+        buildDiscarder(
+            logRotator(
+                numToKeepStr: '3',
+                daysToKeepStr: '30',
+                artifactDaysToKeepStr: '30',
+                artifactNumToKeepStr: '3'
+            )
+        )
+        disableConcurrentBuilds()
+        timeout(time: 60, unit: 'MINUTES')
```

Comment:
You've validated that the entire delete + nested stacks complete w/in 60 minutes? I can't remember whether the per-stage timeouts will effectively override this global timeout.

Reply:
The entire master deploys in short order...under 20 minutes with the delete (remembering to delete the contents of S3)